crates/red_knot_server/src/session/capabilities.rs

```diff
@@ -8,6 +8,8 @@ pub(crate) struct ResolvedClientCapabilities {
     pub(crate) document_changes: bool,
     pub(crate) workspace_refresh: bool,
     pub(crate) pull_diagnostics: bool,
+    /// Whether `textDocument.declaration.linkSupport` is `true`
+    pub(crate) declaration_link_support: bool,
```

Comment:
I think this should be `type_definition_link_support`? Refer to my other comment.